src/components/User/SystemAssets/components/AssetList.vue

```diff
@@ -171,4 +222,10 @@ function convertPolicyToString (policy) {
   width: 10%;
   text-align: right;
 }
+
+.app__more-btn-wrp--left {
```

Comment:
even if the styles scoped, you should no add modifiers to more global scopes than the present one. You can define your own `block__element` here with the needed `block__element--modifier`, or you can just add `block__element` that will act like modifier of `app__more-btn-wrp`, like `app__more-btn-wrp.block__element`